py/surveyqa/summary.py

```diff
@@ -395,6 +395,25 @@ def exptime_dgb(string, color):
     fig.legend.click_policy="hide"
     return fig
 
+def get_moonplot(exposures, width=500, height=300):
+    p = bk.figure(plot_width=width, plot_height=height, x_axis_label = "MOONFRAC", y_axis_label = "MOONALT")
```

Comment:
Please add a docstring clarifying which columns of exposures are required to be able to make this plot.